### PR TITLE
Added endpoint for deassigning tests

### PR DESCRIPTION
**Pull request by Anlord241**

### CODE REVIEW

**Comment by Anlord241** [comment]
What is wrong with checks?

**Comment by maxim12321** [comment]
https://www.githubstatus.com/
GitHub actions failed a bit :)

**Review by AntonMaksimenkau — APPROVED** [review]

File: src/main/java/com/team4/testingsystem/services/impl/TestsServiceImpl.java [hunk 1]
```diff
@@ -113,6 +113,16 @@ public long assignForUser(long userId, Levels levelName, LocalDateTime deadline)
         return test.getId();
     }
 
+    @Override
+    public void deassign(long id) {
```
Comment (maxim12321) [inline src/main/java/com/team4/testingsystem/services/impl/TestsServiceImpl.java:117]: Shouldn't we update the test status after deassign?

Comment (AntonMaksimenkau) [reply src/main/java/com/team4/testingsystem/services/impl/TestsServiceImpl.java:117]: No. the status should not be `assigned` at the point

**Review by maxim12321 — APPROVED** [review]

**Pull request merged**